causely/manifest.json

```diff
@@ -0,0 +1,44 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "0ac3b5eb-46d2-4091-a001-e95dc5782609",
+  "app_id": "causely",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Causal AI platform automatically captures, represents and analyzes causality.",
+    "title": "Causely",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Kubernetes Service Congested Causality",
+        "image_url": "images/Service_Congested_Causality.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Application DB Connection Noisy Neighbor Causality",
+        "image_url": "images/DB_Connections_Causality.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Containers",
+      "Category::Kubernetes",
+      "Category::Incidents",
+      "Offering::Integration",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
```

Comment:
Add relevant data types from this [list](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags)
ex: 

 "Queried Data Type::<FILL IN>"

Reply:
done